datastore/datastore-core/build.gradle

```diff
@@ -15,22 +15,89 @@
  */
 
 import androidx.build.Publish
-import org.jetbrains.kotlin.gradle.tasks.KotlinCompile
+import org.jetbrains.kotlin.gradle.plugin.KotlinPlatformType
+import org.jetbrains.kotlin.gradle.tasks.KotlinNativeCompile
+import org.jetbrains.kotlin.gradle.plugin.mpp.KotlinNativeTarget
 
 plugins {
     id("AndroidXPlugin")
-    id("kotlin")
+    id("org.jetbrains.kotlin.multiplatform")
 }
 
-dependencies {
-    api(libs.kotlinStdlib)
-    api(libs.kotlinCoroutinesCore)
-    api("androidx.annotation:annotation:1.1.0")
+def datastoreType = Attribute.of('androidx.datastore.type', String)
 
-    testImplementation(libs.junit)
-    testImplementation(libs.kotlinCoroutinesTest)
-    testImplementation(libs.truth)
-    testImplementation(project(":internal-testutils-truth"))
+kotlin {
+    jvm {
+        attributes.attribute(datastoreType, 'okio')
```

Comment:
discussed offline but putting it here for reference so we can followup.
my understanding was that, if we declare an attirbute for the "Default", this will require anyone downstream to declare that (and make gradle fail with a confusing error because it cannot find matching artifact).